plutus-core/plutus-ir/test/transform/inline/callsite-non-trivial-body.golden

```diff
@@ -48,37 +43,29 @@
   )
   [
     [
-      addInteger
+      (builtin addInteger)
       [
-        [ [ f (lam unused (con integer) (con integer 5)) ] addInteger ]
+        [
+          [ f (lam unused (con integer) (con integer 5)) ] (builtin addInteger)
+        ]
         (con integer 6)
       ]
     ]
     [
+      [ (builtin addInteger) (con integer 11) ]
```

Comment:
@michaelpj this is the new test result and I can't figure out why it's doing this. If you look at [the output file](https://github.com/input-output-hk/plutus/blob/3c9e8f7716dfbc848ce5e2685d56903661ec0f62/plutus-core/plutus-ir/test/transform/inline/callsite-non-trivial-body.golden) it doesn't seem right.